packages/inventory-insights/src/Insights.js

```diff
@@ -404,7 +404,7 @@ class InventoryRuleList extends Component {
                     actionsConfig={{ actions }}
                     bulkSelect={bulkSelect}
                     filterConfig={{ items: filterConfigItems }}
-                    pagination={<React.Fragment> {results === 1 ? `${results} recommendation` : `${results} recommendations`} </React.Fragment>}
+                    pagination={<React.Fragment> {results === 1 ? `${results} recommendation` : `${results} Recommendations`} </React.Fragment>}
```

Comment:
Plural has capital R and singular has lower r, shouldn't this be the same?

Reply:
yup good call